### ISSUE TITLE
[Feature Request] Better lyrics matching

### ISSUE THREAD

**Issue by lushuyu** [digimezzo/dopamine-windows#1051]
## I'm submitting a...

<pre><code>
[   ] Regression 
[   ] Bug report  
[ x ] Feature request
</code></pre>

There are many lyrics matching API in Dopamine.

But I think they are not good enough at searching Chinese or Japanese lyrics.Netease's music is losing.And Xiami feels like nothing.

I highly recommend QQ music.On songs without copyright, QQ's music just can't be played and lyrics can be obtained.But Netease just deletes them anyway.So maybe QQ's lyrics API is a better choice.

I fonud some information about QQ's API.

https://github.com/ccchangkong/article/issues/23

https://github.com/Rain120/qq-music-api

**Comment by lushuyu** [comment]
Don't mind my poor English. :-)

**Comment by Magentaize** [comment]
You're right, Netease had removed lyric api. But maintain a usable api is a boring business due to their war of copyright. :)

**Comment by lushuyu** [comment]
> You're right, Netease had removed lyric api. But maintain a usable api is a boring business due to their war of copyright. :)

So I choose Dopamine instead of Netease.My playlist has almost been empty because of copyright.（
